inlang/source-code/sdk2/src/mock/bundle.ts

```diff
@@ -17,35 +17,14 @@ export const pluralBundle: BundleNested = {
 				{
 					type: "input",
 					name: "numProducts",
-					value: {
-						type: "expression",
-						arg: {
-							type: "variable",
-							name: "numProducts",
-						},
-					},
```

Comment:
The removed code is an example of the confusion that this PR addresses. An input with name `numProducts` is declaring a variable. The author thought that in addition of an input `numProducts`, the expression of declaring a new `variable` `numProduct` is needed. 

- an input declares a variable, the expression doesn't do anyhting
- the expression references a variable, `type: variable` is not a declaration!